boards/riscv/esp32c3_devkitm/Kconfig.board

```diff
@@ -5,4 +5,8 @@
 
 config BOARD_ESP32C3_DEVKITM
 	bool "ESP32C3 Devkit-M Board"
-	depends on SOC_ESP32C3
+	depends on SOC_SERIES_ESP32C3
```

Comment:
Part of those changes (`SOC_ESP32C3` -> `SOC_SERIES_ESP32C3`) were already done in previous commit ("soc: xtensa: esp32xx: refactor folder structure"). Please keep them together.

Reply:
Ok fixed.